yastd/dbus_api.md

```diff
@@ -0,0 +1,29 @@
+
+## get the list of available languages
+
+Iface: o.o.YaST.Installer1.Language
+
+stupid api:
+  AvailableLanguages -> array(string)
+
+minimal working api:
+  needed: human readable name (how localized?), identifier
```

Comment:
I also expect some setter/getter for selected language ( maybe languages to allow install more locales variants? )

Reply:
Yes
- SelectedLanguages
- DefaultLanguage
- 
(maybe Locale is better than Language)